readme/BREAKING_CHANGES.markdown

```diff
@@ -802,4 +802,94 @@ be made available at
 This change provides the latest features offered by the Portlet 3.0
 Specification, which was released in early 2017.
 
+---------------------------------------
+
+### Changed the Dependency for the liferay-util:html-top JSP tag When Implementing Portlet 3.0
+- **Date:** 2018-Jun-07
+- **JIRA Ticket:** LPS-81983
+
+#### What changed?
+
+When implementing Portlet 3.0, the usage of `portal-kernel`'s `StringBundler`
+has been deprecated in favor of Petra's `StringBundler`.
+
+#### Who is affected?
+
+This affects anyone implementing Portlet 3.0 that uses the
+`<liferay-util:html-top>` JSP tag.
+
+#### How should I update my code?
+
+You must add the following dependency in your build file for your JSPs to
+compile successfully:
+
+**build.gradle**:
+
+    dependencies {
+        ...
+        compileOnly project(":core:petra:petra-string")
```

Comment:
@codyhoag You'll have to ask Greg A. if this is the correct syntax for build.gradle files used by customers. It might have a different syntax, I don't know.